src/main/scala/com/codacy/client/bitbucket/util/Implicits.scala

```diff
@@ -12,8 +11,8 @@ object Implicits {
   implicit def enumWrites[E <: Enumeration#Value]: Writes[E] = Writes((e: E) => Json.toJson(e.toString))
 
   implicit def enumReads[E <: Enumeration](e: E): Reads[e.Value] = {
-    Reads.StringReads.map { value => e.values.find(_.toString == value) }.
-      collect(ValidationError("Invalid enumeration value")) { case Some(v) => v }
+    Reads.StringReads.map { value => e.values.find(_.toString == value) }
+      .collect(JsonValidationError("Invalid enumeration value")) { case Some(v) => v }
```

Comment:
@mrfyda did you confirm this collect works like the Scala one? Just to make sure this will not crash

Reply:
The collect or the JsonValidationError?
ValidationError was renamed to ValidationError

Reply:
I was referring to the fact that this collect is not a Scala collect since it receives the validation as the first argument and then the actual body. Just wondering if it acts like a regular body.